### ISSUE TITLE
Find a way to label PRs with the relevant code owners.

### ISSUE THREAD

**Issue by seanpmorgan** [tensorflow/addons#1145]
Currently we assign maintainers for incoming PRs with [CODEOWNERS](https://github.com/tensorflow/addons/blob/master/.github/CODEOWNERS). However, we're limited to tagging users with write access.

Seeing as a majority of our maintainers are submodule and not repository maintainers, it would be nice to devise a strategy for tagging submodule maintainers (even if it's just an @). Possibly would need to be a bot.

**Comment by gabrieldemarmiesse** [comment]
We had a similaire issue in keras-contrib. Maybe we can port some of the code from there: https://github.com/keras-team/keras-contrib/issues/416
